website_sale_wishlist/models/wishlist.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# © 2016 Leonardo Donelli
+# License AGPL-3 or later (http://www.gnu.org/licenses/agpl)
+
+from openerp import api, fields, models
+
+
+class ShopWishList(models.Model):
+    _name = 'wishlist'
+
+    user_id = fields.Many2one('res.users')
+    item_ids = fields.One2many('wishlist.item', 'wishlist_id', string='Items')
+
+    @api.multi
+    def add_product(self, product_id):
```

Comment:
It would be nice if this used recordset input instead - newer API model code tends to pass around Recordsets instead of IDs, so this is ambiguous IMO (even though your naming of the variable indicates that it is an ID, it's also customary to pass around Recordset singletons as `_id` variables 😑 )

Non blocker for sure, just worth noting.